corehq/apps/performance_sms/forms.py

```diff
@@ -0,0 +1,90 @@
+import copy
+from corehq.apps.groups.fields import GroupField
+from django import forms
+from django.utils.translation import ugettext as _
+from corehq.apps.app_manager.fields import ApplicationDataSourceUIHelper
+from corehq.apps.performance_sms import parser
+from corehq.apps.performance_sms.exceptions import InvalidParameterException
+from corehq.apps.performance_sms.models import TemplateVariable, ScheduleConfiguration, SCHEDULE_CHOICES
+from corehq.apps.reports.daterange import get_simple_dateranges
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Submit
+
+
+class PerformanceMessageEditForm(forms.Form):
+    recipient_id = forms.CharField()
+    schedule = forms.ChoiceField(choices=[(choice, choice) for choice in SCHEDULE_CHOICES])
```

Comment:
not a blocker, but it would be good to make the display text for the choices be translatable